show/muxcable.py

```diff
@@ -381,3 +381,36 @@ def eyeinfo(port, target):
     lane_data.append(res)
     click.echo(tabulate(lane_data, headers=headers))
     sys.exit(EXIT_SUCCESS)
+
+@muxcable.command()
+@click.argument('port', required=True, default=None)
+def cableinfo(port):
+    """Show muxcable cable information"""
+
+    if platform_sfputil is not None:
+        physical_port_list = platform_sfputil_helper.logical_port_name_to_physical_port_list(port)
+
+    if not isinstance(physical_port_list, list):
+        click.echo("ERR: Unable to get a port on muxcable port")
+        sys.exit(EXIT_FAIL)
+        if len(physical_port_list) != 1:
+            click.echo("ERR: Unable to get a single port on muxcable")
+            sys.exit(EXIT_FAIL)
+
+    physical_port = physical_port_list[0]
+    import sonic_y_cable.y_cable
+    res = sonic_y_cable.y_cable.get_pn_number_and_vendor_name(physical_port)
+    if res == False or res == -1:
+        click.echo("ERR: Unable to get cable info")
+        sys.exit(EXIT_FAIL)
+    headers = ['pin_number', 'vendor_name']
```

Comment:
This is not a pin number, but rather a part number, or model. Also suggest making heading more human-readable, e.g., `headers = ['Vendor', 'Model']`

I also suggest refactoring `get_pn_number_and_vendor_name()` in a future PR to split into two functions, `get_model()` and `get_vendor()`. Maybe we should do this now, before merging this PR?

Reply:
Yeah, agree Should call two API's for these. 
will raise another PR for this.

Reply:
fixed